### PR TITLE
Created a new footer based on the CSS

### PR DESCRIPTION
**Pull request by plandsman11**
I created a new Footer based on the Figma. It looks like the following and resizes based on percents. The only thing that could need to be changed is that at the moment the footer is in a rectangular background. The CSS has the background being wavy, not sure how to achieve this without just taking a screenshot. 

Screen Shot 2021-04-19 at 5 31 41 PM

### CODE REVIEW

**Comment by plandsman11** [comment]
Also, we need to add functionality to the social media logos. Do we want to include these?

File: frontend/src/components/Footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            <div className="column"> 
+                <div className="contact-num">
+                    <h1 className="Contact">Contact: </h1>
+                    <h2 className="number"> (310) 7887-0970 </h2>
+                </div>
+            </div>
+            <div className="column"> 
+                <p className="middle-info"> &copy; 2019 Happy Hats for Kids in Hospitals</p>
+                <p className="middle-info"> Happy hats For Kids In Hospitals is a California 501(C)(3) Nonprofit Organization</p>
+                <p className="middle-info"> Founded in 1991 by Sheri Schrier</p>
```
Comment (bglossner) [inline frontend/src/components/Footer/Footer.js:17]: What's the middle-info class?

**Comment by plandsman11** [comment]
Removed the social media logos and replaced it with h4i statement
Screen Shot 2021-04-19 at 6 25 02 PM

File: frontend/src/components/Footer/Footer.js (hunk 1 above)
Comment (plandsman11) [reply frontend/src/components/Footer/Footer.js:17]: That is just for formatting the middle column text.

**Review by AnnaReid22 — APPROVED** [review]

**Pull request merged**